packages/addressable/src/utils/eachUnique.ts

```diff
@@ -0,0 +1,18 @@
+import { Subject } from './Subject';
+
+export function eachUnique(sbj: Pick<Subject, 'subscribe'>) {
+  return {
+    subscribe: (fn: (arg: any) => void) => {
+      let oldState: any = {};
```

Comment:
if it is possible to have multiple states, you can save prev states in Set, and check if they are not in the Set,
in this impl. you check only if the new state is diff from the prev state

Reply:
It's the desire behavior, it suppose to emit the delta from previews change property by property
the name is confusing